server/src/test/java/org/elasticsearch/index/translog/TranslogTests.java

```diff
@@ -4000,8 +4000,8 @@ static boolean hasCircularReference(Exception cause) {
         return false;
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/108321")
     public void testDisabledFsync() throws IOException {
+        var translogDir = createTempDir();
```

Comment:
It would be nice for this variable to have a different name, otherwise it shadows the `translogDir` field.